Components/Pages/Welcome.js

```diff
@@ -22,13 +22,11 @@ const Welcome = ({ navigation }) => {
         </Text>
       </AnimatedView>
       <AnimatedView style={styles.container} delay={600} duration={300}>
-        <Button label="Login" onPress={login} full primary />
+        <Button label="Push" onPress={login} full primary />
```

Comment:
This button that takes you to the login page. Why change it from Login?

Reply:
Whoops - changed it back to Login